CONTRIBUTING.md

```diff
@@ -271,9 +275,7 @@ Fixes #123. The footer can contain Fixes messages.
 Any code change should come with corresponding test changes. Especially bug fixes.
 Tests attached to bug fixes should fail before the change and succeed with it.
 
-```sh
-npm test
-```
+Tests can be run either from the shell (`npm test`) or from the browser(`npm start`; `http://localhost:9999/test`). See [Running tests](#running-tests) for more information.
```

Comment:
I think in this section we should just mention `npm test` but linking to the other section and saying there's more info there is a great idea.